mozci/push.py

```diff
@@ -155,7 +155,17 @@ def tasks(self):
         )
 
         def add(data):
-            for task in data:
+            if "header" in data:
+                data["data"] = [
+                    {
+                        field: entry[i]
+                        for i, field in enumerate(data["header"])
+                        if entry[i] is not None
+                    }
+                    for entry in data["data"]
+                ]
+
+            for task in data["data"]:
```

Comment:
rename `data` to `result`, if only of clarity

Reply:
Done with b63faaf.